nixos/modules/services/misc/passopolis.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.passopolis;
+in
+{
+
+  ###### interface
+
+  options = {
+
+    services.passopolis = {
+
+      enable = mkOption {
+        default = false;
+        description = "
+          Enable passopolis password manager service.
+        ";
+      };
+
+      db_url = mkOption {
+        default = "jdbc:postgresql://localhost:5432/passopolis";
```

Comment:
According to that url https://www.petefreitag.com/articles/jdbc_urls/ the underlying library can.
I would then look similar to this example: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/gogs.nix#L116